doc/user/content/concepts/indexes.md

```diff
@@ -35,6 +38,9 @@ on materialized views serve up-to-date results without themselves performing the
 incremental computation. The in-memory results are accessible to queries within
 the cluster, even for queries that do not use the index key(s).
 
+Although indexes preserve monotonicity information, materialized views do
+**not**. As such, for (indexed or non-indexed) materialized views, results are not monotonic.
```

Comment:
There's a more specific thing to say, if it helps, which is that materialized views will not be monotonic. That may be a preservation of the monotonicity properties, or it may not. Materialized views do not .. randomly choose whether a thing will be monotonic, they don't claim a non-monotonic thing will be monotonic. They will be non-monotonic.

Reply:
Also, idk maybe useful but a helpful synonym we use externally sometime is "append-only". A bit less jargon-y.

Reply:
Main concern about the difference between "do not preserve" and "is not monotonic": several other systems "do not preserve" important correctness properties, and require the user to manage them instead. This is different, and not a scary thing that the user now needs to read up on and understand to correctly operate MZ.